docs/source/index.md

```diff
@@ -1,10 +1,47 @@
+
+* [arviz-base](https://arviz-base.readthedocs.io/en/latest/) data related functionality, including converters from different PPLs.
+* [arviz-plots](https://arviz-plots.readthedocs.io/en/latest/) for visual checks built on top of arviz-stats and arviz-base.
+
+We recommend most users install and use all three ArviZ components together through the main ArviZ package, as they're designed to work seamlessly as one toolkit. Advanced users may choose to install components individually for finer control over dependencies.
+
+Note: All statistical functions - whether accessed through the full ArviZ package or directly via ArviZ-stats - are documented here.
+
+
+## Overview
+
+In Modern Bayesian statistics models are usually build and solve using probabilistic programming languages (PPLs) such as PyMC, Stan, NumPyro, etc. These languages allow users to specify models in a high-level language and perform inference using state-of-the-art algorithms like Markov Chain Monte Carlo (MCMC) or Variational Inference (VI). As a result we usually get a posterior distribution, in the form of samples. The posterior distribution has a central role in Bayesian statistics, but other distributions like the posterior and prior predictive distribution are also of interest. And other quantities may be relevant too.
+
+The correct visualization, analysis, and interpretation of these computed data is key to properly answer the questions that motivate our analysis.
+
+When working with Bayesian models there are a series of related tasks that need to be addressed besides inference itself:
+
+* Diagnoses of the quality of the inference
+
+* Model criticism, including evaluations of both model assumptions and model predictions
+
+* Comparison of models, including model selection or model averaging
+
+* Preparation of the results for a particular audience.
+
+We call these tasks exploratory analysis of Bayesian models (EABM). Successfully performing such tasks are central to the iterative and interactive modelling process (See Bayesian Workflow). In the words of Persi Diaconis.
+
+> Exploratory data analysis seeks to reveal structure, or simple descriptions in data. We look at numbers or graphs and try to find patterns. We pursue leads suggested by background information, imagination, patterns perceived, and experience with other data analyses.
+
+The goal of ArviZ is to provide a unified interface for performing exploratory analysis of Bayesian models in Python, regardless of the PPL used to perform inference. This allows users to focus on the analysis and interpretation of the results, rather than on the details of the implementation.
+
 
 (installation)=
 ## Installation
 
-It currently can only be installed with pip:
+For instructions on how to install the full ArviZ package (including `arviz-base`, `arviz-stats` and `arviz-plots`), please refer to the [installation guide](https://arviz-devs.github.io/arviz/stable/getting_started/Installation.html).
```

Comment:
```suggestion
For instructions on how to install the full ArviZ package (including `arviz-base`, `arviz-stats` and `arviz-plots`), please refer to the [installation guide](https://python.arviz.org/en/latest/getting_started/Installation.html).
```

the arviz.github.io is old and redirects to the 0.14 version of the docs (which didn't have the installation guide yet)